### PR TITLE
Improve umbrella organization

### PR DESCRIPTION
**Pull request by manucamejo**
- Move arena messages.proto
- Add command to run all apps
- Minor fixes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.PHONY: start
+
+start:
+	mix deps.get
+	iex -S mix phx.server
```
Comment (Nico-Sanchez) [inline Makefile:5]: It's not needed now, but we're not running the docker-compose files for each app here.

**Comment by Nico-Sanchez** [comment]
I also pushed a commit to fix the `deps` path in game_client config file.

File: apps/arena/Makefile [hunk 2]
```diff
@@ -15,9 +15,9 @@ stop:
 
 generate-ex-protos:
 	protoc \
-		--elixir_out=lib/arena/protobuf \
-		--elixir_opt=package_prefix=arena.protobuf \
-		messages.proto
+		--elixir_out=lib/arena \
+		--elixir_opt=package_prefix=arena.serialization \
+		serialization/messages.proto
```
[the remark this answers is not in the document]
Comment (manucamejo) [reply apps/arena/Makefile]: Fixed

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (manucamejo) [reply Makefile]: Fixed

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,31 @@
-.PHONY: start
+.PHONY: start generate-protos generate-arena-protos generate-game-client-protos
```
Comment (Nico-Sanchez) [inline Makefile:1]:
```suggestion
.PHONY: start format generate-protos generate-arena-protos generate-game-client-protos
```

File: README.md [hunk 4]
```diff
@@ -1,30 +1,56 @@
...
 
-## Apps
+To start all applications, run the following command:
+```
+make start
+```
 
-To start all apps, run:
-`make start`
+Then navigate to the following link http://localhost:3000/board/1
```
Comment (Nico-Sanchez) [inline README.md:11]:
```suggestion
Then navigate to the following link to start a game: http://localhost:3000/board/1
```

Comment (manucamejo) [reply README.md]: Fixed

File: Makefile [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
 
 start:
+	cd apps/game_client/assets && npm install
 	mix deps.get
 	iex -S mix phx.server
+
+format:
+	cd apps/arena && mix format
+	cd apps/game_client && mix format
```
[the remark this answers is not in the document]
Comment (manucamejo) [reply Makefile]: Fixed

File: README.md [hunk 6]
```diff
@@ -1,30 +1,56 @@
...
+
+### Future iterations
+In future iterations, we will add the following apps:
+
+- Bots
+- Matchmaking
+- Marketplace
+- Chat
+- Inventory
+- Leaderboard
```
Comment (Nico-Sanchez) [inline README.md:54]: Should we mention something about multiplayer? We now have it hardcoded for 1 player but we could change that or say somehting about it.

Comment (manucamejo) [reply README.md:54]: Maybe we can add it in the next iteration of this readme

**Review by Nico-Sanchez — APPROVED** [review]

**Pull request merged**